plugins/Installation/FormDatabaseSetup.php

```diff
@@ -41,6 +41,9 @@ function init()
         $adapters = array();
         foreach ($availableAdapters as $adapter => $port) {
             $adapters[$adapter] = $adapter;
+            if ($adapter == 'PDO\MYSQL') {
+                $adapters[$adapter] .= ' (Recommended)';
```

Comment:
This would ideally use a translation. Can you try something like `'(' . Piwik::translate('General_Recommended') . ')'`?